Auth0/WebAuth.swift

```diff
@@ -138,11 +138,25 @@ public protocol WebAuth: Trackable, Loggable {
      */
     func parameters(_ parameters: [String: String]) -> Self
 
+    /// Setup the response types to be used for authentcation
+    ///
+    /// - Parameter response: Array of ResponseOptions
+    /// - Returns: the same OAuth2 instance to allow method chaining
```

Comment:
All existing documentation in this file references 'OAuth2', I'll rename them all then.